remote-config/build.gradle

```diff
@@ -1,4 +1,5 @@
 apply from: '../config/android/library.gradle'
+apply from: '../config/documentation/dokka/android.gradle'
```

Comment:
I think this requires some additional setup after updating to the new version, right? There needs to be a `dokka` block which configures the source directory: https://github.com/rakutentech/android-buildconfig/tree/master/documentation#dokka

Reply:
Added setting of source directory